x/ethbridge/keeper/msg_server.go

```diff
@@ -312,15 +312,64 @@ func (srv msgServer) RescueCeth(goCtx context.Context, msg *types.MsgRescueCeth)
 		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, msg.CosmosSender)
 	}
 
-	if err := srv.Keeper.ProcessRescueCeth(ctx, msg); err != nil {
-		logger.Error("keeper failed to process rescue ceth message.", errorMessageKey, err.Error())
+	if err := srv.Keeper.ProcessRescueNativeToken(ctx, msg); err != nil {
+		logger.Error("keeper failed to process rescue native_token message.", errorMessageKey, err.Error())
 		return nil, err
 	}
-	logger.Info("sifnode emit rescue ceth event.",
+	logger.Info("sifnode emit rescue native_token event.",
 		"CosmosSender", msg.CosmosSender,
 		"CosmosSenderSequence", strconv.FormatUint(account.GetSequence(), 10),
 		"CosmosReceiver", msg.CosmosReceiver,
-		"CethAmount", msg.CethAmount)
+		"NativeTokenAmount", msg.NativeTokenAmount)
 
-	return &types.MsgRescueCethResponse{}, nil
+	return &types.MsgRescueNativeTokenResponse{}, nil
+}
+
+func (srv msgServer) SetNativeToken(goCtx context.Context, msg *types.MsgSetNativeToken) (*types.MsgSetNativeTokenResponse, error) {
```

Comment:
update function name to match the new message name.

Reply:
renamed